ui/components/ui/loading-screen/index.scss

```diff
@@ -1,7 +1,7 @@
 .loading-overlay {
   top: 0;
   left: 0;
-  z-index: 51;
+  z-index: 55;
```

Comment:
The new red button added above the three buttons in the header to indicate unread notifications, when present, supersedes the stack. That’s why I modified this z-index